src/AutoMerge/dependency_confusion.jl

```diff
@@ -24,26 +21,33 @@ function has_no_dependency_confusion(pkg, registry_head, public_registries)
             registry_toml = TOML.parsefile(joinpath(registry, "Registry.toml"))
             packages = registry_toml["packages"]
             if haskey(packages, uuid)
-                message = string("UUID $uuid conflicts with the package ",
-                                 packages[uuid]["name"], " in registry ",
-                                 registry_toml["name"], " at $repo. ",
-                                 "This could be a dependency confusion attack.")
+                message = string(
+                    "UUID $uuid conflicts with the package ",
+                    packages[uuid]["name"],
+                    " in registry ",
+                    registry_toml["name"],
+                    " at $repo. ",
+                    "This could be a dependency confusion attack.",
+                )
                 # Conflict detected. This is benign if the package name
                 # *and* the package URL matches.
                 if packages[uuid]["name"] != pkg
                     return false, message
                 end
                 package_path = packages[uuid]["path"]
-                other_package_repo = TOML.parsefile(joinpath(registry, package_path, "Package.toml"))["repo"]
+                other_package_repo = TOML.parsefile(
+                    joinpath(registry, package_path, "Package.toml")
+                )["repo"]
                 if package_repo != other_package_repo
                     return false, message
                 end
             end
         catch
-            message = string("Failed to clone public registry $(repo) for a check against dependency confusion.\n",
-                             "This is an internal issue with the AutoMerge process and has nothing to do with ".
-                             "the package being registered but requires manual intervention before AutoMerge ",
-                             "can be resumed.")
+            message = string(
+                "Failed to clone public registry $(repo) for a check against dependency confusion.\n",
+                "This is an internal issue with the AutoMerge process and has nothing to do with "."the package being registered but requires manual intervention before AutoMerge ",
```

Comment:
It's a safe bet that these code lines have never run. I'll grant it that the auto-formatting did something good in this case.